packages/entities/entities-plugins/src/components/fields/KeyAuthIdentityRealms.vue

```diff
@@ -111,7 +123,7 @@ const onRealmsUpdate = (currentSelected: string[]) => {
     ? currentKonnectRealmSelection.filter((realmId) => !prevKonnectRealmSelection.includes(realmId))
     : currentKonnectRealmSelection
 
-  const nextKonnectRealms = nextKonnectRealmId.map((id) => ({ scope: 'realm', id } as const))
+  const nextKonnectRealms = nextKonnectRealmId.map((id) => ({ ...guessRegion(), scope: 'realm', id } as const))
```

Comment:
Do you think if we can invoke this once instead of every item?

Reply:
Good catch! I didn't realize it before.
Luckily, the `nextKonnectRealmId` typically carries only 1 item, so it's safe to write like this here.